pkg/plugins/discovery.go

```diff
@@ -22,41 +22,39 @@ type Registry interface {
 	Plugin(name string) (*Plugin, error)
 }
 
-type LocalRegistry struct {
-	path string
-}
-
-func newLocalRegistry(path string) *LocalRegistry {
-	if len(path) == 0 {
-		path = defaultLocalRegistry
-	}
+type LocalRegistry struct{}
 
-	return &LocalRegistry{path}
+func newLocalRegistry() LocalRegistry {
```

Comment:
i know it's unrelated to this PR, but it's unfortunate we have another thing called Registry that has nothing to do with a docker registry :( we should consider renaming it to `PluginService` or `PluginStore` what do you think? Can be done in another PR, but just wanted to bring it to the attention